parsl/app/bash.py

```diff
@@ -150,6 +150,7 @@ def __call__(self, *args, **kwargs):
                              executors=self.executors,
                              cache=self.cache,
                              ignore_for_cache=self.ignore_for_cache,
-                             app_kwargs=invocation_kwargs)
+                             app_kwargs=invocation_kwargs,
+                             app_type='bash')
```

Comment:
This app_type annotation is not used any more, right? (and I hope to persuade you it will never be needed...)